src/graphics/Screen.cpp

```diff
@@ -949,7 +952,26 @@ bool deltaToTimestamp(uint32_t secondsAgo, uint8_t *hours, uint8_t *minutes, int
     validCached = true;
     return validCached;
 }
-
+#if defined(M5STACK_COREBASIC) || defined(M5STACK_CORE2) 
+static void drawLoraMessage(OLEDDisplay *display, OLEDDisplayUiState *state, int16_t x, int16_t y){
+    display->clear();
+    display->drawString(x + 0, y + 0, myRegion->name);
+    display->setFont(FONT_MEDIUM);
+    display->drawString(x + 10, y + 30,"channel: "+String(DataRegion.lora_channel_num+1));
+    display->drawString(x + 10, y + 60,"bw: "+ String(DataRegion.lora_bw));
+    display->drawString(x + SCREEN_WIDTH/2+10, y+30,"freq: "+String(DataRegion.lora_freq));
+    display->drawString(x + SCREEN_WIDTH/2+10, y+60,"power: "+ String(DataRegion.lora_power_output));
+    display->drawString(x + 10, y + 90,"sf: "+String(DataRegion.lora_sf));
+     display->drawString(x + SCREEN_WIDTH/2+10, y+90,"cr: 4/:"+ String(DataRegion.lora_cr));
+    display->setTextAlignment(TEXT_ALIGN_LEFT);
+    const char *title = "LoRa";
+    display->drawString(x + SCREEN_WIDTH/2-20, y +0, title);
+    display->setFont(FONT_SMALL);
+    display->setTextAlignment(TEXT_ALIGN_RIGHT);
+    display->drawString(x + SCREEN_WIDTH, y,String(DataRegion.lora_channel_name));
+    display->setTextAlignment(TEXT_ALIGN_LEFT); // Restore left align, just to be kind to any other unsuspecting code
+}
+#endif
 /// Draw the last text message we received
```

Comment:
Again please don't introduce new features that are macro-guarded for just one platform. This will work for any device and belongs in a separate PR.

Reply:
ok, I'll make it public